.github/workflows/dev_build.sh

```diff
@@ -72,6 +72,22 @@ check_all.sh
 DREDD_ROOT=`pwd`
 cp ${DREDD_ROOT}/temp/build-Debug/src/dredd/dredd ${DREDD_ROOT}/third_party/clang+llvm-13.0.1/bin/dredd
 
+# examples/simple/pi.cc: check that we can build the simple example
+${DREDD_ROOT}/third_party/clang+llvm-13.0.1/bin/dredd examples/simple/pi.cc
+clang++ examples/simple/pi.cc -o examples/simple/pi
+./examples/simple/pi
```

Comment:
Could you also capture the output of running the `pi` program and use `diff` or something like that to check that it is as expected?

Reply:
Done